config/core/test-configs.yaml

```diff
@@ -1860,6 +1866,13 @@ test_configs:
     test_plans:
       - baseline
 
+  - device_type: at91sam9g20ek
+    test_plans:
+      - baseline
```

Comment:
Tested OK:
https://lava.sirena.org.uk/scheduler/job/36824
https://staging.kernelci.org/test/plan/id/623c40306b1bb198125208fb/

Reply:
Tested OK (again, with the fix to drop `arch: armel`):
https://lava.sirena.org.uk/scheduler/job/47254
https://staging.kernelci.org/test/plan/id/6246b66fa57b0ef47b05f4c6/